doc/links_names.inc

```diff
@@ -80,6 +80,7 @@
 .. _numpy: http://numpy.scipy.org
 .. _scipy: http://www.scipy.org
 .. _ipython: http://ipython.scipy.org
+.. _IPython: http://www.ipython.org/
```

Comment:
This line generates a conflict with the line above.  Can you just remove it ? 

Thanks